docs/tutorial/part-one/index.md

```diff
@@ -36,12 +38,11 @@ recommended version for your operating system.
 ## Install the "Hello World" starter
 
 Gatsby uses "starters" for starting new projects. Starters
-are partially built Gatsby sites pre-configured to help you get moving faster.
+are partially built Gatsby sites that are pre-configured to help you get moving faster.
 There are several official starters and many others contributed from the Gatsby
 community! [See the Starters page for the full list](/docs/gatsby-starters/).
 
-To install a starter, first install Gatsby's terminal program by typing the
-following command into the terminal window and hit ENTER.
+To install a starter, first install Gatsby's terminal program by running the following command:
```

Comment:
Should be `CLI` not terminal. Or actually "first install Gatsby's command line program"